CHANGELOG.md

```diff
@@ -6,7 +6,10 @@ description: |-
 ---
 
 # VKCS Provider's changelog
+
 #### v0.7.3 (unreleased)
+- Add ability to search by extra_specs param of vkcs_compute_flavor
+- Enhancement searching the closest appropriate flavor by min_ram and min_disk argument
```

Comment:
Improve filtering in vkcs_compute_flavor data source: select the closest appropriate flavor when either "min_ram" or "min_disk" is specified.

Reply:
too long
and we do not want write anything about min_disk

Reply:
mix your comments and fix